client/js/src/main/scala/org/http4s/client/oauth1/oauth1platform.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2014 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s
+package client
+package oauth1
+
+import cats.effect.Async
+import cats.syntax.all._
+import org.http4s.js.webcrypto.subtle
+import scala.scalajs.js
+import scala.scalajs.js.typedarray._
+import scala.scalajs.js.typedarray.TypedArrayBuffer
+import org.http4s.js.crypto.KeyFormat
+import org.http4s.js.crypto.KeyUsage
+import org.http4s.js.crypto.KeyAlgorithm
+import java.nio.charset.StandardCharsets
+import org.http4s.js.crypto.CryptoKey
+import java.util.Base64
+import org.http4s.js.crypto.BufferSource
+
+private[oauth1] trait oauth1platform {
```

Comment:
Thanks for making these private.  We didn't do this on our literal macros, and it made me sad earlier today.

Reply:
Nit: the name makes me twitch.  Can we capaitalize it like a regular trait: `Oauth1Platform`?

Reply:
Yup, easy fix :) tried my best to remember to mark platforms as private, let me know if I missed any!